.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
         run: |
           COMMIT_DETAILS=$(echo "${COMMIT_DETAILS}" | sed 's/\\:/:/g')
```

Comment:
It looks like the author is trying to modify the content of `RELEASE_NOTES` by removing certain characters and replacing some patterns using `sed` commands. It would be good to review the logic behind these modifications to ensure they are correct and necessary.